tests/common/devices.py

```diff
@@ -404,6 +404,26 @@ def get_critical_group_and_process_lists(self, container_name):
                 succeeded = False
                 break
 
+        # For PMon container, since different daemons are enabled in different platforms, we need find common processes
+        # which are not only in the critical_processes file and also are running in that platform.
+        if succeeded and container_name == "pmon":
+            common_groups = []
+            common_processes = []
+            process_list = self.shell("docker exec {} supervisorctl status".format(container_name))
+            for process_info in process_list["stdout_lines"]:
+                process_name = process_info.split()[0].strip()
+                process_status = process_info.split()[1].strip()
+                if ":" in process_name:
```

Comment:
Is this check necessary? Do we need to maintain backward-compatibility with the old syntax? The syntax has been updated in 201911+ to have the colon, correct?

Reply:
At here, this `if` statement is used to check whether the `process_name` retrieved from the command `supervisorctl status` is in the format `group_name:critical_process`. If it has such format, then the `group_name` in it will be got and checked to see whether it is in `critical_group_list`. If it is not, then `else` statment will check whether `process_name` is in the `critical_process_list`.

Reply:
But with the new syntax, all lines should have a colon, either preceded by `group` or `process`. There should no longer be a need for the 'else' case here.

Reply:
Sorry, I was thinking this was the critical_processes file parsing. Nevermind.

Reply:
No problem!